.travis.yml

```diff
@@ -9,4 +9,17 @@ services:
 before_install:
   - bundle install
 
-script: bundle exec kitchen verify
+
+env:
+  matrix:
+    - INSTANCE: default-ubuntu-1804
+    - INSTANCE: default-debian-stretch
+    - INSTANCE: default-debian-jessie
+    - INSTANCE: default-centos-7
+    - INSTANCE: version-1122-ubuntu-1804
```

Comment:
Hey I think these names need to start version-18031 now. Right? If you all choose to keep this. This is one reason I let Test Kitchen just control the test matrix.

Reply:
Yep! See the errors in Travis:

`No instances for regex `version-1122-ubuntu-1804', try running `kitchen list'`